conf/addons/nvidia-test.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "nvidia-test"
+spec:
+  restartPolicy: "OnFailure"
+  nodeSelector:
+    beta.kubernetes.io/instance-type: "p2.xlarge"
+  tolerations:
+  - key: "nvidia.com/gpu"
+    effect: "NoSchedule"
+  containers:
+    - name: "cuda-vector-add"
+      # https://github.com/kubernetes/kubernetes/blob/v1.7.11/test/images/nvidia-cuda/Dockerfile
+      image: "k8s.gcr.io/cuda-vector-add:v0.1"
```

Comment:
not sure if this image will ever change, but maybe
{{ getenv "CUDA_VECTOR_ADD_IMAGE" | default "k8s.gcr.io/cuda-vector-add:v0.1" }}